apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/TransformerConstants.java

```diff
@@ -138,6 +138,8 @@
    */
   public static final String CODING_SYSTEM_HCPCS = BASE_URL_BBAPI_RESOURCES + "/codesystem/hcpcs";
 
+  public static final String CODING_SYSTEM_CPT = "http://www.ama-assn.org/go/cpt";
```

Comment:
we won't need this once we start implementing the 0.1+ versions.  All of our proc codes will be hcpcs